### ISSUE TITLE
PageRank, meta discussion

### ISSUE THREAD

**Issue by jowens** [eunaeuna/GraphFrameworkOnGPUs#11]
I find PageRank to be a pretty useless benchmark these days because the common way it is reported—always a dense vector, fixed number of iterations—means that it boils down to SpMV and pretty much every framework implements it the same way with the same performance. In other words, it's just not a graph benchmark that matters any more since everyone has the same performance.

(This is also likely why Gunrock defaults are fine, because there's really only one way to implement this.)

**Comment by ogreen** [comment]
Agreed. One of the reasons that this algorithm\problem is especially important is the fact that it can be implemented in so many different ways. GAS, BLAS, vertex centric. 
We have at least two different implementations. This could be useful information for the GraphBLAS community.

**Comment by jowens** [comment]
But what the text in the paper seemed to indicate was that we're all doing it one way (dense vector, measuring time per iteration).
